tests/suites/casper/click.js

```diff
@@ -51,7 +52,7 @@ casper.test.begin('clickLabel tests tests', 12, function(test) {
         var results = this.getGlobal('results');
         if (phantom.casperEngine === 'slimerjs') {
             // "javascript:" link in Gecko are executed asynchronously, so we don't have result at this time
-            test.skip(1);
+            test.skip(1, 'Casper.click() on javascript: skipped for slimer engine');
```

Comment:
I wonder if we couldn't have some `skipTestIfEngine(test, engine)` reusable helper to avoid duplicating code for these.